.travis.yml

```diff
@@ -1,2 +1,7 @@
 languages: java
+jdk:
+  - openjdk7
```

Comment:
Unfortunately, Travis only supports oraclejdk8 for now and has not added support for openjdk8 yet. Adding openjdk8 to this config results in an error.